game/scripts/vscripts/libraries/timers.lua

```diff
@@ -195,68 +180,62 @@ function Timers:HandleEventError(name, event, err)
 	end
 end
 
-function Timers:CreateTimer(name, args, context)
-	if type(name) == "function" then
-		if args ~= nil then
-			context = args
+function Timers:CreateTimer(arg1, arg2, context)
+	if type(arg1) == "function" then
+		if arg2 ~= nil then
+			context = arg2
 		end
-		args = {callback = name}
-		name = DoUniqueString("timer")
-	elseif type(name) == "table" then
-		args = name
-		name = DoUniqueString("timer")
-	elseif type(name) == "number" then
-		args = {endTime = name, callback = args}
-		name = DoUniqueString("timer")
+		arg2 = {callback = arg1}
+	elseif type(arg1) == "table" then
+		arg2 = arg1
+	elseif type(arg1) == "number" then
+		arg2 = {endTime = arg1, callback = arg2}
 	end
-	if not args.callback then
-		print("Invalid timer created: "..name)
+	if not arg2.callback then
+		print("Invalid timer created: "..arg1)
```

Comment:
Since there's no such thing as timer's name there's nothing to show there (though it wasn't really useful before). print + return can be replaced with
```lua
error("Invalid timer created")
```

Reply:
done